src/Features/OnboardingTasks/Task.php

```diff
@@ -323,7 +289,7 @@ public function undo_snooze() {
 	 */
 	public function has_previously_completed() {
 		$complete = get_option( self::COMPLETED_OPTION, array() );
-		return in_array( $this->id, $complete, true );
+		return in_array( $this->get_id(), $complete, true );
 	}
 
 	/**
```

Comment:
The `can_view` and `is_complete` uses should be changed to functions in the below function.
Getting a `PHP Notice:  Undefined property` on line 299